paper-api/src/main/java/org/bukkit/block/Lidded.java

```diff
@@ -1,25 +1,34 @@
 package org.bukkit.block;
 
+/**
+ * @deprecated Incomplete api. Use {@link io.papermc.paper.block.Lidded} instead.
+ */
+@Deprecated // Paper - Deprecate Bukkit's Lidded API
```

Comment:
TODO: remove this lone `// paper` comment
```suggestion
@Deprecated
```